app/components/datafruits-player.hbs

```diff
@@ -0,0 +1,130 @@
+<audio preload="" id="radio-player" class="hidden">
```

Comment:
Did you mean to move this template from `app/templates/components` to `app/components` ??

Reply:
Yeah, but I think that was unrelated, and probably doesn't need to happen in this PR :sweat_smile: